app/Services/EntityService.php

```diff
@@ -594,6 +594,31 @@ public function getEnabledEntities(Campaign $campaign, $except = [])
         return $entityTypes;
     }
 
+    /**
+     * @param array $except
+     * @return array
+     */
+    public function getEnabledEntitiesSorted(Campaign $campaign, bool $singular = true, $except = []): array
```

Comment:
Rather than have a parameter for campaign, you can use `->campaign($campaign)->getEnabledEntitiesSorted()`